prepare/tasks/regression/tasks.py

```diff
@@ -13,6 +13,7 @@
         prediction_type="Any",
         metrics=["metrics.spearman"],
         augmentable_inputs=["text"],
+        defaults={"attribute_name": "similarity"},
```

Comment:
While similarity is a common regression, we can assume it's the default, because there are others..

Reply:
I think that if we want to assume that we should specify "similarity" in the task name

Reply:
We can "tasks.regression.two_texts.similarity".